uasc/secure_channel.go

```diff
@@ -124,7 +124,7 @@ func (s *SecureChannel) SendRequest(req ua.Request, authToken *ua.NodeID, h func
 func (s *SecureChannel) SendRequestWithTimeout(req ua.Request, authToken *ua.NodeID, timeout time.Duration, h func(interface{}) error) error {
```

Comment:
this belongs to a different patch

Reply:
I have no idea how this is contaminating my branch. I even redid this PR from scratch.